### PR TITLE
Fix promoted tspan for BVProblem and ODEProblem

### PR DESCRIPTION
**Pull request by ErikQQY**

### CODE REVIEW

**Comment by ChrisRackauckas** [comment]
This needs a test.

File: src/ensemble/ensemble_problems.jl [hunk 1]
```diff
@@ -16,7 +16,11 @@ DEFAULT_REDUCTION(u, data, I) = append!(u, data), false
 DEFAULT_VECTOR_PROB_FUNC(prob, i, repeat) = prob[i]
 function EnsembleProblem(prob::AbstractVector{<:AbstractSciMLProblem}; kwargs...)
     # TODO: @invoke
-    invoke(EnsembleProblem, Tuple{Any}, prob; prob_func=DEFAULT_VECTOR_PROB_FUNC, kwargs...)
+    invoke(EnsembleProblem,
+        Tuple{Any},
+        prob;
+        prob_func = DEFAULT_VECTOR_PROB_FUNC,
+        kwargs...)
```
Comment (ChrisRackauckas) [inline src/ensemble/ensemble_problems.jl:23]: What's up with this formatting? This doesn't look like the right formatter.

Comment (ChrisRackauckas) [reply src/ensemble/ensemble_problems.jl:23]: @isaacsas do you know what could be causing this?

Comment (isaacsas) [reply src/ensemble/ensemble_problems.jl:23]: The formatting? No idea. There weren’t any recent changes, and the last change was for function declaration signatures only right?

Comment (isaacsas) [reply src/ensemble/ensemble_problems.jl:23]: Maybe this was a manual reformatting that rejected the auto format?

Comment (ChrisRackauckas) [reply src/ensemble/ensemble_problems.jl:23]: @ErikQQY can you please minimize the change here and do the formatting in a different PR?

**Review by ChrisRackauckas — APPROVED** [review]

File: src/ensemble/ensemble_problems.jl (hunk 1 above)
Comment (ErikQQY) [reply src/ensemble/ensemble_problems.jl]: done

**Pull request merged**